pkgs/misc/emulators/wine/staging.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, callPackage, wineUnstable }:
+{ lib, stdenv, callPackage, wineUnstable }:
```

Comment:
The other `lib` in this file are used as `stdenv.lib`, so do we want it as an extra argument for `concatMapStringsSep`?

Reply:
ok, will use `stdenv.lib` as well